### ISSUE TITLE
Error: The RxJS instance does not seem to provide an `Observable` type.

### ISSUE THREAD

**Issue by dottodot** [feathersjs-ecosystem/feathers-reactive#81]
This might be because I'm trying to use buzzard but thought I'd check as not had this issue on other projects. In my angular service I have 

```
import { Injectable } from '@angular/core';
import * as feathers from '@feathersjs/client';
import * as feathersRx from 'feathers-reactive';
import * as io from 'socket.io-client';
import * as socketio from '@feathersjs/socketio-client';
import * as authentication from '@feathersjs/authentication-client';
import { environment } from '@env/environment';
import { CookieStorage } from 'cookie-storage';

@Injectable()
export class FeathersService {
  private _feathers: any;
  private _socket: any;
  constructor() {
    this._socket = io(environment.host);
    this._feathers = feathers();
    this._feathers.configure(socketio(this._socket));
    this._feathers.configure(
      feathersRx({
        idField: '_id'
      })
    );

    this._feathers.configure(
      authentication({
        storageKey: 'feathers-token',
        storage: new CookieStorage({
          path: '/',
          expires: new Date(Date.now() + 30 * 24 * 60 * 60 * 1000)
        })
      })
    );
  }
}
```

but this is giving an error of

> core.js:1350 ERROR Error: Uncaught (in promise): Error: The RxJS instance does not seem to provide an `Observable` type.
> Error: The RxJS instance does not seem to provide an `Observable` type.
>     at FeathersRx (index.js:33)
>     at new FeathersService (feathers.service.ts:18)
>     at _createClass (core.js:10620)
>     at _createProviderInstance$1 (core.js:10596)
>     at resolveNgModuleDep (core.js:10581)
>     at NgModuleRef_.get (core.js:11806)
>     at resolveDep (core.js:12302)
>     at createClass (core.js:12166)
>     at createDirectiveInstance (core.js:12011)
>     at createViewNodes (core.js:13449)
>     at FeathersRx (index.js:33)
>     at new FeathersService (feathers.service.ts:18)
>     at _createClass (core.js:10620)
>     at _createProviderInstance$1 (core.js:10596)
>     at resolveNgModuleDep (core.js:10581)
>     at NgModuleRef_.get (core.js:11806)
>     at resolveDep (core.js:12302)
>     at createClass (core.js:12166)
>     at createDirectiveInstance (core.js:12011)
>     at createViewNodes (core.js:13449)
>     at resolvePromise (zone.js:824)
>     at eval (zone.js:876)
>     at ZoneDelegate.invokeTask (zone.js:425)
>     at Object.onInvokeTask (core.js:4620)
>     at ZoneDelegate.invokeTask (zone.js:424)
>     at Zone.runTask (zone.js:192)
>     at drainMicroTaskQueue (zone.js:602)
>     at <anonymous>

if I remove feathers reactive the error goes away.

**Comment by j2L4e** [comment]
Pretty sure this is caused by feathers reactive being pinned to rxjs 5.4 at the moment. You can try the rxjs5.5 branch of the repo 

Am 19. November 2017 19:06:30 MEZ schrieb Marcus Williams <notifications@github.com>:
>This might be because I'm trying to use buzzard but thought I'd check
>as not had this issue on other projects. In my angular service I have 
>
>```
>import { Injectable } from '@angular/core';
>import * as feathers from '@feathersjs/client';
>import * as feathersRx from 'feathers-reactive';
>import * as io from 'socket.io-client';
>import * as socketio from '@feathersjs/socketio-client';
>import * as authentication from '@feathersjs/authentication-client';
>import { environment } from '@env/environment';
>import { CookieStorage } from 'cookie-storage';
>
>@Injectable()
>export class FeathersService {
>  private _feathers: any;
>  private _socket: any;
>  constructor() {
>    this._socket = io(environment.host);
>    this._feathers = feathers();
>    this._feathers.configure(socketio(this._socket));
>    this._feathers.configure(
>      feathersRx({
>        idField: '_id'
>      })
>    );
>
>    this._feathers.configure(
>      authentication({
>        storageKey: 'feathers-token',
>        storage: new CookieStorage({
>          path: '/',
>          expires: new Date(Date.now() + 30 * 24 * 60 * 60 * 1000)
>        })
>      })
>    );
>  }
>}
>```
>
>but this is giving an error of
>
>> core.js:1350 ERROR Error: Uncaught (in promise): Error: The RxJS
>instance does not seem to provide an `Observable` type.
>> Error: The RxJS instance does not seem to provide an `Observable`
>type.
>>     at FeathersRx (index.js:33)
>>     at new FeathersService (feathers.service.ts:18)
>>     at _createClass (core.js:10620)
>>     at _createProviderInstance$1 (core.js:10596)
>>     at resolveNgModuleDep (core.js:10581)
>>     at NgModuleRef_.get (core.js:11806)
>>     at resolveDep (core.js:12302)
>>     at createClass (core.js:12166)
>>     at createDirectiveInstance (core.js:12011)
>>     at createViewNodes (core.js:13449)
>>     at FeathersRx (index.js:33)
>>     at new FeathersService (feathers.service.ts:18)
>>     at _createClass (core.js:10620)
>>     at _createProviderInstance$1 (core.js:10596)
>>     at resolveNgModuleDep (core.js:10581)
>>     at NgModuleRef_.get (core.js:11806)
>>     at resolveDep (core.js:12302)
>>     at createClass (core.js:12166)
>>     at createDirectiveInstance (core.js:12011)
>>     at createViewNodes (core.js:13449)
>>     at resolvePromise (zone.js:824)
>>     at eval (zone.js:876)
>>     at ZoneDelegate.invokeTask (zone.js:425)
>>     at Object.onInvokeTask (core.js:4620)
>>     at ZoneDelegate.invokeTask (zone.js:424)
>>     at Zone.runTask (zone.js:192)
>>     at drainMicroTaskQueue (zone.js:602)
>>     at <anonymous>
>
>if I remove feathers reactive the error goes away.
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/feathersjs-ecosystem/feathers-reactive/issues/81

-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.

**Comment by dottodot** [comment]
I not sure if is it that as I just inadvertently installed feathers-reactive@next and that still depends on  5.4.x but doesn't give the error.

**Comment by daffl** [comment]
I think this should also be fixed via #95
